packages/lesswrong/components/ea-forum/onboarding/useEAOnboarding.tsx

```diff
@@ -40,7 +19,7 @@ type EAOnboardingContext = {
 }
 
 const eaOnboardingContext = createContext<EAOnboardingContext>({
-  currentStage: onboardingStages[0],
+  currentStage: "",
```

Comment:
arguably this default value is a bit sus, I think alternative approach would be to make it nullable, but that would maybe involve having to change the existing stages to accommodate that?

Reply:
I think making it an index would be ideal, in the previous version it was fulfilling the role of defaulting to "the first stage" and it just so happened that we could get the actual stage name because it was hardcoded. Failing that though I think null would be better

Reply:
will make it nullable!